fmriprep/workflows/bold/outputs.py

```diff
@@ -77,8 +78,9 @@ def prepare_timing_parameters(metadata):
     ...     prepare_timing_parameters(dict(RepetitionTime=2, SliceTiming=[0.0, 0.2, 0.4, 0.6]))
     {'RepetitionTime': 2, 'SliceTimingCorrected': False, 'DelayTime': 1.2}
     >>> with mock.patch("fmriprep.config.workflow.ignore", ["slicetiming"]):
-    ...     prepare_timing_parameters(dict(VolumeTiming=[0.0, 1.0, 2.0, 5.0, 6.0, 7.0],
-    ...                                    SliceTiming=[0.0, 0.2, 0.4, 0.6, 0.8]))
+    ...     prepare_timing_parameters(
+    ...         dict(VolumeTiming=[0.0, 1.0, 2.0, 5.0, 6.0, 7.0],
+    ...              SliceTiming=[0.0, 0.2, 0.4, 0.6, 0.8]))  #doctest: +NORMALIZE_WHITESPACE
```

Comment:
Should we just add a pytest section to `pyproject.toml`?

```
[tool.pytest]
doctest_optionflags = NORMALIZE_WHITESPACE
```

Reply:
That won't impact testing the installed version with `--pyargs` without pulling the repository in. We do have that in `setup.cfg` already.